src/components/BottomBarTab.vue

```diff
@@ -1,11 +1,9 @@
 <template>
-  <div class="bottombartab" :style="{ background: `#${color}` }" :class="{ inactive: !isFirstTab}" @click="bottomBarTabToggle(courseObj)">
-    <div class="bottombartab-wrapper" @click="toggleFromTab">
-      <img v-if="isFirstTab && !isExpanded" class="bottombartab-arrow" src="@/assets/images/uparrow-white.svg"/>
-      <img v-if="isFirstTab && isExpanded" class="bottombartab-arrow" src="@/assets/images/downarrow-white.svg"/>
+  <div class="bottombartab" :style="{ background: `#${color}` }" :class="{ inactive: !isBottomCourseFocus}" @click="bottomBarTabToggle(courseObj)">
+    <div class="bottombartab-wrapper">
       <div class="bottombartab-name">{{subject}} {{number}}</div>
     </div>
-    <img class="bottombartab-delete" src="@/assets/images/x-white.svg" @click="deleteBottomTab(subject, number)"/>
+    <img class="bottombartab-delete" src="@/assets/images/x-white.svg" @click.stop="deleteBottomTab(courseObj)"/>
```

Comment:
Haven't seen this .stop before and struggling to understand from the docs but am curious. What does this do?

Reply:
Haha I looked it up too. It's a shorthand for calling `e.stopPropagation()`, which stops the click event from propagating up to any parents. https://flaviocopes.com/vue-events/#event-modifiers

So in this case, if I want to delete a bottom bar tab (by clicking the x-icon), it won't call the function normally called from clicking the tab itself (`bottomBarTabToggle(courseObj)`).
You can actually observe this bug in production right now; if you try to delete a tab besides the first tab, it switches the focus to another course.